src/test/java/org/folio/edge/patron/utils/PatronOkapiClientCompressionTest.java

```diff
@@ -67,17 +65,18 @@ public void testCompression(TestContext context) throws Exception {
     headers.add("Accept-Encoding", "gzip");
     Async async = context.async();
     client.getAccount(patronId,
-        true,
-        true,
-        true,
+      true,
+      true,
+      true,
+      null,
+      null,
+      null,
```

Comment:
Why are these parameters indented differently to those below?

Reply:
looks like I removed whitespace before method arguments, Returned it back